src/include/cache.h

```diff
@@ -202,14 +202,15 @@ struct __wt_cache {
 	uint64_t las_remove_count;      /* Count of removes from lookaside */
 	uint64_t las_pageid;		/* Lookaside table page ID counter */
 
-	bool las_reader;		/* Indicate an LAS reader to sweep */
+	bool las_reader;		/* Indicate a LAS reader to sweep */
```

Comment:
In my head I say the letter names, not "lass". So in that case `an ELL` is correct.